viniciuslima-99/introduction-to-web-development
# Hotfix/files-in-folders

### What I did

- Changed the README.md file
- Added the pages folder to place html files
- Added the style folder to place css files
- Added the scripts folder to place javascript files
- Added the image folder to place image files

### How to test

1. Execute the index.html file in the pages folder and make sure that the entire project flow, style and scripts is working.
